source/sync/data-model/update-data-model/update-schema-production-apps.txt

```diff
@@ -1,20 +1,430 @@
+
+            3. Your changes are applied to the new schema version. The new
+               version is compatible with previous versions.
+
+         .. tab:: Breaking
+            :tabid: unsupported
+
+            If your changeset includes at least one breaking change, then all
+            changes (including any non-breaking and schema versioning changes)
+            are applied in a single changeset. Upon deployment of breaking
+            changes, App Services terminates and re-initializes Device Sync.
+
+            1. Review your changes, which may include non-breaking and schema
+               versioning changes. We recommend that you resolve any suggested
+               actions noted before you deploy your changes.
+
+               Breaking changes may or may not increment the schema version:
+
+               - If the current schema version is greater than 0, App Services
+                 indicates that the schema will increment upon deployment.
+               - If the current schema version is 0, the window only displays
+                 the current schema version number.
+
+            2. Click :guilabel:`Deploy and re-initialize`.
+
+            3. After Device Sync terminates and re-initializes, your changes are
+               applied in one of two ways:
+
+               - If App Services created a new schema, it becomes the new
+                 minimum version and is incompatible with any previous schemas.
+               - If App Services did not create a new schema, the changes are
+                 applied to the current schema version 0.
+
+   .. step:: Handle Client Resets
+
+            After you deploy your changes, you must do the following to ensure
+            that clients can successfully connect to Atlas and sync data with
+            the new schema:
+
+            .. tabs::
+
+               .. tab:: Schema Versioning
+                  :tabid: supported
+
+                  For schema versioning changes, App Services automatically
+                  handles the client reset and migration for you.
+
+                  To ensure App Services is able to perform the migration
+                  successfully, you must open synced database on the
+                  client *asynchronously*. This blocks the client from writing
+                  to the synced database until after the client reset is
+                  complete.
+
+                  For more information, refer to the following SDK
+                  documentation:
+
+                  .. TODO: add link to SDK docs
+
+                  - C++ SDK
+                  - Flutter SDK
+                  - Java SDK
+                  - Kotlin SDK
+                  - .NET SDK
+                  - Node.js SDK
+                  - React Native SDK
+                  - Swift SDK
+
+               .. tab:: Breaking
+                  :tabid: unsupported
+
+                  For breaking changes, you *must* perform a client
```

Comment:
I might be wrong here, but it seems like you have written here that _both_ cases involve a client reset. But I believe that in the case of sync terminating and re-enabling, the sync history is incompatible so the changes are unrecoverable and this would be a "manual" client reset - i.e. unsynced data is lost. Probably worth verifying if that's accurate, and then maybe clarifying those details.